pyproject.toml

```diff
@@ -11,7 +11,7 @@ PyYAML = "^6.0"
 mysqlclient = {version = "^2.1.1", optional = true}
 SQLAlchemy-Utils = "^0.38.3"
 requests = "^2.28.1"
-synapseclient = {version = "^2.6.0", optional = true}
+synapseclient = "^2.7.0"
```

Comment:
@andrewelamb is it possible to keep `synapseclient` as `optional` or should we reconfigure the dependency for schematic?